src/components/libraryoptionseditor/libraryoptionseditor.js

```diff
@@ -410,6 +410,11 @@ import template from './libraryoptionseditor.template.html';
         } else {
             parent.querySelector('.chkEnableEmbeddedEpisodeInfosContainer').classList.add('hide');
         }
+        if (contentType === 'movies') {
+            parent.querySelector('.chkAutoCollectionContainer').classList.remove('hide');
+        } else {
+            parent.querySelector('.chkAutoCollectionContainer').classList.add('hide');
+        }
```

Comment:
I don't insist, but using `toggle` is shorter.
```suggestion

        parent.querySelector('.chkAutoCollectionContainer').classList.toggle('hide', contentType !== 'movies');
```
_empty line to separate blocks_

Reply:
Thank you, fixed that.